server/src/controllers/questionsController.ts

```diff
@@ -313,3 +314,29 @@ export const createCommentHandler = async (req: Request, res: Response) => {
       .json({ error: "Error Adding Your Comment to the Database" });
   }
 };
+
+export const editQuestionHandler = async (req: Request, res: Response) => {
+  const questionId = Number(req.params.id);
+
+  if (!questionId) {
+    res.status(400).json({ error: "Question Id was not provided" });
+  }
+
+  const question = req.body.questionContent;
```

Comment:
```suggestion
  const question = req.body.questionContent;
  logger.info({
    message: "editQuestionHandler question",
    value: question
  });
```
we need to add logger here